pantheon-bundle/frontend/src/app/searchResults.tsx

```diff
@@ -20,6 +21,7 @@ import {
 } from "@patternfly/react-core";
 import { SearchIcon } from "@patternfly/react-icons";
 import CheckCircleIcon from "@patternfly/react-icons/dist/js/icons/check-circle-icon"
+// import ExclamationTriangleIcon from "@patternfly/react-icons/dist/js/icons/exclamation-circle-icon"
```

Comment:
will we be using this component or can we delete this line

Reply:
deleted.